qemu/tests/virtio_fs_bad_socket_group_name.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+import shutil
+import re
+
+from virttest import data_dir
+from virttest import error_context
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Negative test.virtiofsd shouldn't boot up with an unknown socket group name.
+    Steps:
+        1. Create shared directories on the host.
+        2. Run virtiofsd daemons with an unknown socket group name.
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    result_pattern = params.get("result_pattern", "unable to find group")
+
+    # set fs daemon path
+    fs_source = params.get('fs_source_dir')
+    base_dir = params.get('fs_source_base_dir', data_dir.get_data_dir())
+
+    if not os.path.isabs(fs_source):
+        fs_source = os.path.join(base_dir, fs_source)
+    if os.path.exists(fs_source):
+        shutil.rmtree(fs_source, ignore_errors=True)
+    logging.info("Create filesystem source %s.", fs_source)
+    os.makedirs(fs_source)
```

Comment:
Need to clean up the environment at the end

Reply:
Updated,thx